README.md

```diff
@@ -19,12 +19,12 @@ Tag names are all lower case, so a test named "tagFoo" has a tag "tagfoo".
 
 To run only those tests which have been tagged with a specified tag `tagFoo`:
 ```
-sbt "test-only * -- -n tagfoo"
+sbt "testOnly * -- -n tagfoo"
 ```
 
 Or to instead exclude all tests which have been tagged with a specified tag `tagFoo`:
 ```
-sbt "test-only * -- -l tagfoo"
+sbt "testOnly * -- -l tagfoo"
```

Comment:
Via http://www.scala-sbt.org/0.13/docs/ChangeSummary_0.13.0.html#camelCase+Key+names:

> The convention for key names is now camelCase only instead of camelCase for Scala identifiers and hyphenated, lower-case on the command line. camelCase is accepted for existing hyphenated key names and the hyphenated form will still be accepted on the command line for those existing tasks and settings declared with hyphenated names. Only camelCase will be shown for tab completion, however.